docu/release_notes/2016-10-28-Technev150.html.md

```diff
@@ -0,0 +1,21 @@
+---
+date: October 28, 2016
+service: 'hybris techne'
+official_version: v1.5.0
+headline: 'Techne v1.5.0 released, New Product Thumbnail component. Revisited Tree pattern.'
+---
+
+<h1>Techne v1.5.0</h1>
+<p>New Product Thumbnail pattern added. Can be used to show a list of products in a grid.</p>
+<p>New Grid Started pages added. Speeds the development of pages used to display products in a grid. It has also our new toolbar for easy filtering and navigation.</p>
+<p>Revisited Tree pattern, along with respective Started page. Now with more standard expand/collapse icons and behavior. Description of how the interactivity should work.</b>
+
+<h3>Minor updates</h3>
+<ul>
+    <li>Improved Icons page now displaying the icons in a more easy to glance grid. Added suggestions on how to use colors on icons</li>
```

Comment:
Improved Icons page with grid layout for easy scanability. Added suggested colors for each icon based on it's recommended usage